knpuniversity/oauth2-client-bundle/1.20/config/packages/knpu_oauth2_client.yaml

```diff
@@ -0,0 +1,16 @@
+knpu_oauth2_client:
+    # configure your clients as described here: https://github.com/knpuniversity/oauth2-client-bundle#configuration
+    clients:
+        # # will create a service: "knpu.oauth2.client.facebook_main"
+        # first, run `composer require league/oauth2-facebook`
+        # facebook_main:
+        #     # this will be one of the supported types
+        #     type: facebook
+        #     client_id: '%env(FB_ID)%'
+        #     client_secret: '%env(FB_SECRET)%'
+        #     # the route that you're redirected to after
+        #     # see the controller example below
+        #     redirect_route: connect_facebook_check
+        #     # route parameters to pass to your route, if needed
+        #     redirect_params: {}
+        #     graph_api_version: v2.12
```

Comment:
I think we should ONLY include the link, but *not* include all of the `facebook_main` and details below that. Those details could become out of date, and unless you are using Facebook exactly, they won't be useful.

Reply:
Will update this, let's hope we get it through, as the guidelines say that there shouldn't be 'empty' configs.